lib/tictactoe/cli/console.ex

```diff
@@ -0,0 +1,50 @@
+defmodule TicTacToe.CLI.Console do
+  def display_instructions do
+    IO.puts("Welcome to Tic Tac Toe!\n")
+    IO.puts(instructions())
+    IO.puts("Starting new game...\n")
+  end
+
+  def display_board(board) do
+    length = row_length(board.cells)
+
+    board.cells
+    |> Enum.chunk_every(length)
+    |> Enum.map_join(horizontal_line(length), &format_row(&1))
+    |> IO.puts()
+  end
+
+  defp row_length(cells) do
+    cells
+    |> length()
+    |> :math.sqrt()
+    |> trunc()
+  end
+
+  defp format_row(row_array) do
+    row_array
+    |> Enum.map_join("|", &format_cell(&1))
+  end
+
+  defp format_cell(cell) do
+    " #{cell} "
+  end
+
+  defp horizontal_line(length) do
+    separator =
+      List.duplicate("---", length)
+      |> Enum.join("|")
+
+    "\n#{separator}\n"
+  end
+
+  defp instructions do
```

Comment:
What do you think about making this into a [module attribute](https://elixir-lang.org/getting-started/module-attributes.html)?